admin/src/main/resources/io/buoyant/admin/js/dashboard.js

```diff
@@ -42,9 +43,9 @@ $.when(
   var routers = Routers(metricsJson[0], metricsCollector);
 
   var buildVersion = $(".server-data").data("linkerd-version");
-  var procInfo = ProcInfo(metricsCollector, $(".proc-info"), Handlebars.compile(overviewStatsRsp[0]), buildVersion);
-  var requestTotals = RequestTotals(metricsCollector, $(".request-totals"), Handlebars.compile(requestTotalsRsp[0]), _.keys(metricsJson[0]));
-  var routerDisplays = RouterController(metricsCollector, selectedRouter, routers, routerTemplates, $(".dashboard-container"));
+  var _procInfo = ProcInfo(metricsCollector, $(".proc-info"), Handlebars.compile(overviewStatsRsp[0]), buildVersion);
```

Comment:
since we don't actually use these names we could just avoid naming them and do 
`ProcInfo(metricsCollector, $(".proc-info"), Handlebars.compile(overviewStatsRsp[0]), buildVersion);`